ChainLink-Server/graphql/resolvers/friendship.js

```diff
@@ -112,8 +113,54 @@ module.exports = {
             } catch (err) {
                 throw new Error(err);
             }
-        }
-        
+        },
+        async getInvitableFriends(_, { username, eventID }, context) {
+            try {
+                const friendships = await friendship.find({
+                    $or: [
+                        { sender: username, status: 'accepted' },
+                        { receiver: username, status: 'accepted' },
+                    ]
+                });
+                let friends = friendships.map(friendship => {
+                    if (friendship.sender === username) {
+                        return friendship.receiver;
+                    } else {
+                        return friendship.sender;
+                    }
+                });
+
+                // There is probably a pure mongo way to do this, but I won't be the one to figure that out.
+
+                //let friendObjs = [];
+
+                const friendObjs = await Promise.all(friends.map(async (friend) => 
+                    await users.findOne({username: friend})
+                ));
+
+                friends = [];
+                event = await events.findOne({_id: eventID})
+
+                console.log(friendObjs);
```

Comment:
Can you remove the console log here too please? clutters up the feed when looking at things in prod